css-size-adjust-1/Overview.bs

```diff
@@ -97,54 +108,90 @@ The default size adjustment affects text and form controls, whether
 
-  <li>when the objects to be adjusted are inside a block-level or
+  <li>When the objects to be adjusted are inside a block-level or
   ''display: inline-block'' element with a 'height' other than
-  ''height/auto'' (see [[!CSS21]]),</li>
+  ''height/auto'' (see [[!CSS21]]). This is a signal that the elements
+  are structural to the page content.</li>
 
-  <li>when the objects to be adjusted are inside a
+  <li>When the objects to be adjusted are inside a
   ''display: inline-block'' element
   with a 'width' other than ''width/auto'' (see
-  [[!CSS21]]),</li>
+  [[!CSS21]]). This is a signal that the elements
+  are structural to the page content.</li>
 
-  <li>when the objects to be adjusted have 'white-space' of ''white-space/pre'' or
+  <li>When the objects to be adjusted have 'white-space' of ''white-space/pre'' or
   ''white-space/nowrap'' (see [[!CSS21]]) or a 'text-wrap' of ''text-wrap/nowrap'' (see
   [[!CSS-TEXT-4]]).</li>
 
+  <li>Editability of an element, either because it's a form element or because it is ''contenteditable''</li>
+
+  <li>Element contents; empty elements might not benefit from being autosized</li>
+
+  <li>Elements which have compute styles that are fairly unique among the page. These are often structural
+  and might provide more broken layout than they benefit from size adjustment</li>
+
+  <li>Elements lay out to be extremely wide compared to the viewport. These would cause horizontal scrolling
+  even on a desktop machine</li>
+
+  <li>Elements which are extremely narrow. These are usually structural</li>
+
+  <li>Elements which only contain whitespace</li>
 </ul>
 
-Calculation of default adjustment {#default-adjustment-calculation}
--------------------------------------------------------------------
+Calculation of adjustment {#default-adjustment-calculation}
+-----------------------------------------------------------
+
+A User Agent may consider any information when determining the magnitude
+of adjustment.
 
-The adjustment performed should be based on preferences (of the
-renderer or the renderer's user) indicating the desired <dfn>minimum
-readable text size</dfn>.
-Given this preference, for each containing block of text to be adjusted,
-there is a <dfn>minimum block text size</dfn>:  the preference for the
-minimum readable text size, times the width of the containing block,
-divided by the width of the device.
+Note: In general, User Agents are trying to determine how big to make an
+element so that it is readable, and how small it has to be to not overflow
+or get clipped by other nearby elements.
 
-The size adjustment involves multiplication of sizes by a ratio
-determined by the minimum block text size and the computed value of
-'font-size'.  This ratio must be at least the first divided by the
-second; however, in order to maintain differentiations between font
-sizes, it should often be slightly larger.  <span class="issue">Define
-this with more detail/precision.</span>
```

Comment:
I think it's probably worth retaining this text as an example, and perhaps adding another example if you have one.  It may also be worth adding the "differentiations between font sizes" point to the list of criteria.